indexers/keyvalue/MongoDBIndexer/__init__.py

```diff
@@ -34,43 +37,68 @@ def post_init(self):
                                       password=self.password,
                                       database=self.database_name,
                                       collection=self.collection_name)
-    
+
     def get_add_handler(self):
+        """Get the database handler.
+        """
         return self.handler
 
     def get_create_handler(self):
+        """Get the database handler.
+        """
         return self.handler
 
     def get_query_handler(self):
+        """Get the database handler.
+        """
         return self.handler
-    
+
     def add(self, keys: Iterator[int], values: Iterator[bytes], *args, **kwargs) -> None:
+        """Add JSON-friendly serialized documents to the indexer.
+
+        :param keys: document ids
+        :param values: JSON-friendly serialized documents
```

Comment:
Then the type hint isnt a little missleading? are bytes a correct hint?

Reply:
thanks I changed it to `serialized documents`